modules/alloy/src/main/ts/ephox/alloy/toolbar/Overflows.ts

```diff
@@ -1,8 +1,13 @@
-import { Arr, Fun, Option, Struct } from '@ephox/katamari';
+import { Arr, Fun, Option } from '@ephox/katamari';
 
 import * as PositionArray from '../alien/PositionArray';
 
-const output = Struct.immutable('within', 'extra', 'withinWidth');
+// TODO: what types are these? I think it's AlloyComponent[], AlloyComponent, number
+const output = (within: any, extra: any, withinWidth: number) => ({
```

Comment:
I missed this earlier sorry, but this would be `<T>(within: T[], extra: T, withinWidth: number)`. Reason being overflow allows anything to be passed in, for example the test uses strings instead of AlloyComponents.

Reply:
Ideally `Pos` also should be a generic for the `element` as well, but that's another days issue 👍

Reply:
I've fixed the type of `output`. 
`Pos` looks a little trickier.

Reply:
Yeah, that doesn't surprise me. Thanks for updating `output` though 👍